pre_commit/resources/hook-tmpl

```diff
@@ -40,6 +41,7 @@ fi
 {pre_push}
 
 # Run pre-commit
+args="$args --hook-stage $hook_stage"
```

Comment:
I don't understand why this would happen? Is it because if there are no arguments, then it defaults to run? If that is the case, then why is there an `args` variable in the first place?